helpers.py

```diff
@@ -82,6 +82,7 @@ def cmd(*args, cwd=None, capture=False):
             out += line
     sys.stdout.buffer.write(b'|--\n')
     p.wait()
+    p.communicate()
```

Comment:
interesting - i'm guessing you stumbled across some bug that required this?

Reply:
Yeah, it's just hung in some cases from time to time. Also, it should be enough to have just p.communicate which also waits for the child process termination.